libraries/AP_Common/missing/cmath

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include <toolchain.h>
```

Comment:
We are not talking about a generic include, in which header reorganization, include-what-you-use etc are good. We are talking about a generated header with the compiler defines. It's just like changing the command line for all sources and pretending you shouldn't recompile them.